Update conditionalaccessusers.md (#18080)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/conditionalaccessenumeratedexternaltenants.md

---
title: "conditionalAccessEnumeratedExternalTenants resource type"

> description: "Represents list of external tenants in the policy scope."

Reviewer: ```suggestion
description: "Represents a list of external tenants in the policy scope."
```

ms.localizationpriority: medium
author: "sandeo"
ms.prod: "identity-and-sign-in"
doc_type: resourcePageType
---

# conditionalAccessEnumeratedExternalTenants resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Represents list of external tenants in the policy scope.

Reviewer: ```suggestion
Represents a list of external tenants in the policy scope.
```

## Properties

| Property     | Type        | Description |
|:-------------|:------------|:------------|

> | members | String collection | This represents a collection of tenant ids in scope of Conditional Access for guest and external users policy targeting. |

Reviewer: ```suggestion
| members | String collection | Rrepresents a collection of tenant ids in scope of Conditional Access for guest and external users policy targeting. |
```
